### PR TITLE
Add Python 3.9 to CI

### PR DESCRIPTION
**Pull request by peymanslh**
Changes:
- Add Python 3.9 to TravisCI config file

Closes: #474

### CODE REVIEW

**Comment by WisdomPill** [comment]
There is no python 3.9 yet on travis, python3.9 has been released just 10 days ago and travis still needs to support it.

They only support python3.9-dev as of now.

Please switch to python3.9-dev for this moment and leave it on the allowed to fail builds

**Review by WisdomPill — CHANGES_REQUESTED** [review]
use py39-dev because py39 is not yet available

**Comment by jdufresne** [comment]
What is the reason only "djangomaster" is being tested by Travis? Why not all Django targets?

**Comment by peymanslh** [comment]
> What is the reason only "djangomaster" is being tested by Travis? Why not all Django targets?

I saw Python 3.8 was only tested with `django master`.
We have `django22`, `django30`, `django31`, and `djangomaster` in `tox.ini`.
Should I add all of these for Python `3.9-dev`?

**Comment by WisdomPill** [comment]
> I checked Python 3.8 and it was only tested with django master.

Where have you seen that?

**Comment by hugovk** [comment]
How about adding the Trove classifier to `setup.cfg` too?

```
    Programming Language :: Python :: 3.9
```

File: .travis.yml [hunk 1]
```diff
@@ -56,13 +68,25 @@ matrix:
...
     - env: TOXENV=py37-djangomaster-redislatest
     - env: TOXENV=py37-djangomaster-redismaster
     - env: TOXENV=py38-djangomaster-redislatest
     - env: TOXENV=py38-djangomaster-redismaster
+    - env: TOXENV=py39-dev-django22-redislatest
+    - env: TOXENV=py39-dev-django22-redismaster
+    - env: TOXENV=py39-dev-django30-redislatest
+    - env: TOXENV=py39-dev-django30-redismaster
+    - env: TOXENV=py39-dev-django31-redislatest
+    - env: TOXENV=py39-dev-django31-redismaster
```
Comment (jdufresne) [inline .travis.yml:87]: Thanks for keeping at it!

```suggestion
```

IMO, only `djangomaster` should allow failures, as it is unstable and changing. Python 3.9 is stable and no new PRs should break it.

**Comment by peymanslh** [comment]
@jdufresne Thanks for your review.
I've fixed the conflict with the last commit.

**Review by jdufresne — APPROVED** [review]
LGTM. Thanks!

@WisdomPill Let me know if you'd like to a take a last look.

**Review by WisdomPill — APPROVED** [review]
Nice job, thanks to everybody who contributed

**Pull request merged**